Arashayo/TabletopHaven
# Vue.js setup, login via google, materializecss initialization

## Related Issue
https://app.clickup.com/t/86bwu2nd1
https://app.clickup.com/t/86bx01f6k

## Changes list

- Added login via google
- Vue.js frontend set up
- Implemented Materializecss for vue.js 

## Testing instructions

1. Run npm install in both backend and frontend directories
2. Run backend server
3. Run frontend server (instructions in updated readme)
4. go to http://localhost:8081
5. check if login button is funky (if it is - materializecss loads and works properly)
6. check if login via google with @edu.cdv.pl email leads you to /success subpage
7. 
## Screenshots (if appropriate):
